modules/signing_scriptworker/templates/passwords-mobile.json.erb

```diff
@@ -1,11 +1,6 @@
 {
     "project:mobile:focus:releng:signing:cert:release-signing": [
-        ["signing7.srv.releng.mdc1.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing8.srv.releng.mdc1.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing9.srv.releng.mdc1.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing10.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing11.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing12.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"]
+        ["http://autograph-external.prod.autograph.services.mozaws.net/", "<%= scope.function_secret(["autograph_focus_prod_username"]) %>", "<%= scope.function_secret(["autograph_focus_prod_password"]) %>", ["autograph_focus"], "autograph"]
```

Comment:
@JohanLorenzo I wonder if it's `autograph_apk` instead of `autograph_focus` ?

Reply:
Nevermind, these are all values consumed by scriptworker, hence invisible to Autograph, so they can be whatever we want them to be.

Reply:
Yup, signingscript is the sole consumer of that value.